content/structure.md

```diff
@@ -23,15 +23,42 @@ In ES2015, you can make variables available outside a file using the `export` ke
 
 Since this is a new feature introduced in Meteor 1.3, you will find a lot of code online that uses the older, more centralized conventions built around packages and apps declaring global symbols. This old system still works, so to opt-in to the new module system code must be placed inside the `imports/` directory in your application. We expect a future release of Meteor will turn on modules by default for all code, because this is more aligned with how developers in the wider JavaScript community write their code.
 
-You can read about the module system in detail in the [`modules` package README](https://docs.meteor.com/#/full/modules). This package is automatically included in every new Meteor app as part of the `ecmascript` meta-package, so most apps won't need to do anything to start using modules right away.
+You can read about the module system in detail in the [`modules` package README](https://docs.meteor.com/#/full/modules). This package is automatically included in every new Meteor app as part of the [`ecmascript` meta-package](https://docs.meteor.com/#/full/ecmascript), so most apps won't need to do anything to start using modules right away.
 
-<h3 id="importing-from-packages">Importing from packages</h3>
+<h3 id="intro-to-import-export">Introduction to using `import` and `export`</h3>
 
-In Meteor, it is also simple and straightforward to use the `import` syntax to load npm packages on the client or server, and access the package's exported symbols as you would with any other module. You can also import from Atmosphere packages, but the import path must be prefixed with `meteor/` to avoid conflict with the npm package namespace. For example, to import `HTTP` you could use `import { HTTP } from 'meteor/http'`.
+Meteor allows you to `import` not only JavaScript in your application, but also CSS and HTML to control load order:
 
-<h3 id="using-require">Using `require`</h3>
+```js
+import '../../api/lists/methods.js';  // import from relative path
+import '/imports/startup/client';     // import module with index.js from absolute path
+import './loading.html';              // import HTML from relative path
```

Comment:
This is maybe not quite correct. I'm not sure what the best one liner is, but it's something like "import Blaze-compiled HTML"